### PR TITLE
Sass and less style cleanup and config improvements

### PR DESCRIPTION
**Pull request by bjornalm**
## What changes were proposed in this pull request?

- Ant and cuix styles properly married without leaking into legacy Hue UI
- Improved less setup/configuration of ant design. Got rid of unneeded complexity.  **No need to import individual component styles form Ant to the root-wrapped-antd.less anymore**
- Removed colors.scss since we now get those from cuix
- Renamed hue hue5-gobals.scss to partial _classes.scss 
- Turned the mixins and variables files in to sass "partials". 
- Removed the webpack module replacement for `root-wrapped-antd.less`. It was a leftover from an older integration attempt and not needed since we actually import this file from hue.less
-  Included `desktop/core/src/desktop/js/components/styles` as new sass path so that the partials _classes, _mixins and _variables can be easily imported from anywhere.
- Updated the tsx ReactExample with some more components to verify the styling
- Updated the contribute-frontent instructions
- Updated all references to Sass variables starting with `$fluid-` to `$fluidx-` since that is how they are named in cuix.

## How was this patch tested?

- Manual testing of ReactExample component

Please review [Hue Contributing Guide](https://github.com/cloudera/hue/blob/master/CONTRIBUTING.md) before opening a pull request.

### CODE REVIEW

**Review by JohanAhlen — APPROVED** [review]
LGTM!

File: desktop/core/src/desktop/js/reactComponents/FileChooser/FileChooserModal/FileChooserModal.scss [hunk 1]
```diff
@@ -14,15 +14,15 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-@import '../../../components/styles/colors';
+@use 'variables' as vars;
```
[the remark this answers is not in the document]
Comment (bjornalm) [reply desktop/core/src/desktop/js/reactComponents/FileChooser/FileChooserModal/FileChooserModal.scss:17]: No need, we could use `variables`. `vars` is just slightly shorter and familiar from many examples so I used that. Do you want me to change it?

[the remark this answers is not in the document]
Comment (JohanAhlen) [reply desktop/core/src/desktop/js/reactComponents/FileChooser/FileChooserModal/FileChooserModal.scss:17]: In some places you do `vars.$fluidx-gray-050` in others just `$fluidx-gray-050` just wondering if there was a conflict requiring the `vars` qualifier?

[the remark this answers is not in the document]
Comment (bjornalm) [reply desktop/core/src/desktop/js/reactComponents/FileChooser/FileChooserModal/FileChooserModal.scss:17]: Ahh, I see. In the the Vue related styles I've just used find and replaced so we are still importing with old fashion "import" statements. In the newer react related sass files we import with `use` and then we need a namespace, which by default is the name of the file (variables) or anything we name it using `as ...`.

**Review by nidhibhatg — APPROVED** [review]
LGTM!

File: desktop/core/src/desktop/js/reactComponents/FileChooser/FileChooserModal/FileChooserModal.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (JohanAhlen) [reply desktop/core/src/desktop/js/reactComponents/FileChooser/FileChooserModal/FileChooserModal.scss:17]: Got it. So `@use` requires the qualifier... Then I'd say go for `vars` throughout.

**Pull request merged**